docs/guide/faq/index.md

```diff
@@ -2,6 +2,14 @@
 
 [[toc]]
 
+## Mocking Dates and Timers with Vitest
+
+`mount` depends on the system time in some way \[needs clarification\]. For
```

Comment:
Do you intend to clarify? If not, we could make this more generic. For example:

"Some edge cases are known to exist when using Vitest and the `vi.setSystemTime` that can interfere with Vue's scheduler, causing unexpected behaviors. If you are using `setSystemTime`, make sure you call it *before* calling `mount` to avoid any unexpected behavior".

Reply:
Is that better? I'm not clarifying because I don't know anything and don't plan to find out. But you pretty much just clarified. And I paraphrased it.